xpsi/utilities/ProjectionTool.py

```diff
@@ -280,21 +281,46 @@ def check_model_param_names():
         ##### OUTPUTS #####
         hot_name: name of primary hot spot (according to our naming convention)
         hot_name_s: name of the secondary hot spot (according to our naming convention)
+        hot_name_t: name of the tertiary hot spot (according to our naming convention)
         """
         hot_name = ''
         hot_name_s = ''
+        hot_name_t = ''
         if ('-' in model) and (not('-S' in model) and  not('-U' in model) and not('-Ua' in model)):
-            print (r"ERROR: model not recognised. If model has derived quantities and does not fall into the categories mentioned below: transform your dictionary to match \"-U\" requirements. Recognised hot spot names are \"ST\", \"\PST\", \"\CST\", \"EST\", \"PDT\", \"CDT\", \"EDT\"; for two hot spot model connect each hot spot model with \"+\" (first for primary; second for secondary). If the two hot spots have the same name, add \"-S\" (if all the secondary hot spot is antipodal and its temperature and radius is the same of the primary); \"-U\" if the properties of the secondary spot is completely unrelated to the primary ones; \"-Ua\" if the secondary hot spot is antipodal to the primary but has independent radius and temperature.")
+            print ("ERROR: model not recognised. If model has derived quantities and does not fall into the categories mentioned below: transform your dictionary to match \"-U\" requirements. Recognised hot spot names are \"ST\", \"\PST\", \"\CST\", \"EST\", \"PDT\", \"CDT\", \"EDT\"; for two hot spot model connect each hot spot model with \"+\" (first for primary; second for secondary). If the two hot spots have the same name, add \"-S\" (if all the secondary hot spot is antipodal and its temperature and radius is the same of the primary); \"-U\" if the properties of the secondary spot is completely unrelated to the primary ones; \"-Ua\" if the secondary hot spot is antipodal to the primary but has independent radius and temperature.")
```

Comment:
Why did you remove the letter "r" from here? There should be "r" so that it won't give a SyntaxWarning, since having a string that has backslashes?

Reply:
I added the "r" now back.

Reply:
Sorry I did not remove it, I guess it was already not in the original branch when I pulled to create this one